recipes/spire-pipeline/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "spire-pipeline" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lamyj/spire/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: ce47585d8db70e03a64983dd451f6875adcf6176207abe6bc34f6830f3b87dc6
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - doit
+    - jinja2
+    - numpy x.x
```

Comment:
Why are you trying to pin the numpy version like this? This isn't a compiled package, so why does it matter if the numpy version is the same at packaging/runtime?

Reply:
There is indeed no reason for pinning numpy. Fixed in 0c78896.